AgentDiscoveries-Backend/src/main/resources/config.properties

```diff
@@ -2,7 +2,7 @@
 # Database properties
 database.url = jdbc:mysql://localhost:3306/agentdiscoveries
 database.username = root
-database.password = root
+database.password = password

```

Comment:
Do we want to be committing this? Can't remember what you guys agreed on so feel free to ignore